03/app.js

```diff
@@ -1,31 +1,68 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
+import textarea from './components/styles/textarea';
+import commentli from './components/styles/commentli';
+
 class Article extends React.Component {
     state = {
         comments: [],
+        textarea: ''
+    }
+
+    /* nasłuch */
+    submitEl = (e) => {
+        e.preventDefault();
+        this.addCommentToList(e);
+    }
+
+    /* tekst komentarza */
+    textComment = (e) => {
+        this.setState({textArea: e.target.value})
+    }
+
+    /* komentarz dodany do listy komentarzy */
+    addCommentToList = (e) => {
+        const {textArea} = this.state;
+        if (textArea !== '' && textArea !== undefined) {
```

Comment:
Jak sprawdzasz czy coś jest `undefined` to lepiej to zrobić przy pomocy `typeof`